puffin_egui/src/flamegraph.rs

```diff
@@ -711,30 +735,53 @@ fn to_ms(ns: NanoSecond) -> f64 {
     ns as f64 * 1e-6
 }
 
+#[allow(clippy::too_many_arguments)]
 fn paint_scope(
     info: &Info,
     options: &mut Options,
     stream: &Stream,
+    scope_collection: &ScopeCollection,
```

Comment:
you could also put `scope_collection` in `Info` to reduce the number of arguments being passed around